manifest_staging/charts/csi-secrets-store-provider-azure/templates/provider-azure-installer.yaml

```diff
@@ -85,6 +96,11 @@ spec:
             {{- if .Values.linux.volumeMounts }}
             {{- toYaml .Values.linux.volumeMounts | nindent 12 }}
             {{- end}}
+            {{- if  and .Values.Azure.proxySettings.isProxyEnabled .Values.Azure.proxySettings.proxyCert }}
+            - mountPath: /etc/ssl/certs/proxy-cert.crt
+              subPath: proxy-cert.crt
```

Comment:
Without subPath the file is mounted as symlink. This does not work with provider and it times out while making AKV calls. By requesting subPath file is mounted as it without symlink and provider is able to get secrets.  We have added `checksum` annotation to make sure if there is an update to proxy cert then it will be picked up by next helm upgrade and pods will be rotated with new one.